node/resolvers/catalog/utils.ts

```diff
@@ -34,3 +34,9 @@ export const getBrandFromSlug = async (brandSlug: string, {clients:{catalog}}: C
     toLower(catalogSlugify(brand.name)) === brandSlug || toLower(Slugify(brand.name)) === brandSlug
   )
 }
+
+export async function asyncForEach(array: Array<any>, callback: Function) {
+  for (let index = 0; index < array.length; index++) {
+    await callback(array[index], index, array);
```

Comment:
I think this could be done parallelly(sp?), with `Promise.all` for example, instead of waiting on every iteration.
this could be potentially pretty slow when n is large enough

Reply:
Resolved in new commit